### ISSUE TITLE
installation

### ISSUE THREAD

**Issue by rojinsafavi** [GreenleafLab/ArchR#244]
I am trying to install your latest version, and I followed what you have on the main page. But when I load the archer package, it seems to me that it is not the newest version. For example I cannot find cellToUse in the addCoAccessibility

Thank you!
Rojin

**Comment by rcorces** [comment]
Related to https://github.com/GreenleafLab/ArchR/issues/149#issuecomment-654287012
We're behind on documentation (which is my fault) and it looks like we didnt re-build the function parameters before pushing to master so this parameter is available but it wont show up.

**Comment by rojinsafavi** [comment]
thanks :-)

**Comment by rcorces** [comment]
@jgranja24 - do you want to re-build the param documentation and push to `master`?
